### ISSUE TITLE
[Bug] [Jdbc] NullPointerException of jdbc engine connection clickhouse

### ISSUE THREAD

**Issue by casionone** [apache/incubator-linkis#2557]
### Search before asking

- [X] I searched the [issues](https://github.com/apache/incubator-linkis/issues) and found no similar issues.

### Linkis Component

linkis-cg-engineConnplugin

### What happened + What you expected to happen

```
Exception in thread "main" java.lang.NullPointerException
	at java.util.Hashtable.put(Hashtable.java:460)
	at org.apache.linkis.manager.engineplugin.jdbc.ConnectionManager.createDataSources(ConnectionManager.java:140)
	at org.apache.linkis.manager.engineplugin.jdbc.ConnectionManager.main(ConnectionManager.java:311)
```
```
        Properties props = new Properties();
        props.put("driverClassName", supportedDBs.get(dbType));
        props.put("url", url);
        props.put("maxIdle", 5);
        props.put("minIdle", 0);
        props.put("maxActive", 20);
        props.put("initialSize", 1);
        props.put("testOnBorrow", false);
        props.put("testWhileIdle", true);
        props.put("validationQuery", this.supportedDBsValidQuery.get(dbType));
```
`props.put("validationQuery", this.supportedDBsValidQuery.get(dbType));`  put with null 

```
val JDBC_SUPPORT_DBS_VALIDATION_QUERY = CommonVars("wds.linkis.jdbc.support.dbs.validation.query", "mysql=>SELECT 1,postgresql=>SELECT 1,oracle=>SELECT 1 FROM dual,hive2=>SELECT 1,presto=>SELECT 1")
```
for `wds.linkis.jdbc.support.dbs.validation.query`, new dbtype should with default value or  without put `props.put("validationQuery", this.supportedDBsValidQuery.get(dbType))`

### Relevent platform

linkis-1.1.3

### Reproduction script

none

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

**Comment by huapan123456** [comment]
can i  fix it ?

**Comment by casionone** [comment]
> can i fix it ?

The community has already followed up to fix this problem. Welcome to solve other unresolved issues

**Comment by casionone** [comment]
has been fix in  #2196
